crates/re_space_view_spatial/src/space_view_2d.rs

```diff
@@ -215,17 +204,24 @@ impl SpaceViewClass for SpatialSpaceView2D {
                             images_by_bucket
                                 .into_iter()
                                 .map(|(_, entity_bucket)| {
+                                    // Pick a shared parent as origin.
+                                    // Mostly because it looks nicer in the ui.
+                                    let root = entity_bucket.iter().skip(1).fold(
+                                        entity_bucket
+                                            .first()
+                                            .unwrap_or(&EntityPath::root())
+                                            .clone(),
+                                        |acc, e| acc.common_ancestor(e),
+                                    );
```

Comment:
let's make a helper for this

```suggestion
                                    let root = EntitPath::common_ancestor_of(entity_bucket);
```